docs/decisions/0015-default-enrollments.rst

```diff
@@ -0,0 +1,54 @@
+==============================
+Default Enterprise Enrollments
+==============================
+
+Status
+======
+Proposed - October 2024
+
+Context
+=======
+Enterprise needs a solution for managing automated enrollments into "default" courses or specific course runs
+for enterprise learners. The solution needs to support customer-specific enrollment configurations
+without tightly coupling these configurations to a specific subsidy type, i.e. we should be able in the future
+to manage default enrollments via both learner credit and subscription subsidy types.
+
+Core requirements
+-----------------
+1. Internal staff should be able to configure one or more default enrollments with either a course
+   or a specific course run for automatic enrollment. In the case of specifying a course,
+   the default enrollment flow should cause the "realization" of default enrollments for learners
+   into the currently-advertised, enrollable run for a course.
+2. Default Enrollments should be loosely coupled to subsidy type.
+3. Unenrollment: If a learner chooses to unenroll from a default course, they should not be automatically re-enrolled.
+4. Graceful handling of license revocation: Upon license revocation, we currently downgrade the learner’s
+   enrollment mode to ``audit``. This fact should be visible from any new APIs exposed
```

Comment:
Good call to make the course mode known in the proposed `default-enterprise-enrollment-intentions/learner-status/` GET API.